src/librustdoc/clean/types.rs

```diff
@@ -1505,12 +1505,12 @@ impl Type {
         self.primitive_type().is_some()
     }
 
-    crate fn projection(&self) -> Option<(&Type, DefId, Symbol)> {
-        let (self_, trait_, name) = match self {
-            QPath { self_type, trait_, name, .. } => (self_type, trait_, name),
+    crate fn projection(&self) -> Option<(&Type, DefId, PathSegment)> {
+        let (self_, trait_, last) = match self {
```

Comment:
Why change it to `last` and not `assoc`?

Reply:
oh, this one actually got missed when I had originally named the `QPath` member `last` and then renamed it to `assoc`. thanks for catching this.